README.md

```diff
@@ -20,7 +20,7 @@ pip install https://github.com/xuhcc/beancount-exchangerates/archive/master.zip
 
 Source string format is `<quote-currency>:beancount_exchangerates/<base-currency>:<quote-currency>`.
 
-Default API base url is https://api.frankfurter.app. It can be changed using `EXCHANGERATE_API_URL` environment variable to any service that implements Fixer API.
+Default API base url is `https://api.frankfurter.dev/v1/`. It can be changed using `EXCHANGERATE_API_URL` environment variable (which should end with slash) to any service that implements Fixer API.
```

Comment:
>(which should end with slash)

Not necessarily? I think `urljoin` should take care of that.

Reply:
> > (which should end with slash)
> 
> Not necessarily? I think `urljoin` should take care of that.

Nope, I tried, it doesn't work without slash IFF there is something (such as `/v1/`) ... that's because [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) is _"combining a “base URL” (base) with another URL (url)" - and "combining" a "base URL of e.g. <https://api.frankfurter.dev/v1> with e.g. <latest> gives <https://api.frankfurter.dev/latest> instead of <https://api.frankfurter.dev/v1/latest>... which makes sense - that's how such "relative" URLs e.g. on a HTML page would also work.